copier.py

```diff
@@ -152,8 +152,20 @@ def copy_file(self, file_info, overwrite, dryrun, verify_md5=False):
                        }
 
             if dryrun:
+                if verify_md5:  #assume validate file if verify_md5 is true and dryrun is true
+                    if _is_local(org_url): #validate org_file_size against real local file size
```

Comment:
This size validation should always be executed, no matter dryrun or verfify_md5 have been set or not.

Reply:
Updated, thanks!